opencti-platform/opencti-front/lang/back/en.json

```diff
@@ -585,6 +592,7 @@
   "Subject public key exponent": "Subject public key exponent",
   "Subject public key modulus": "Subject public key modulus",
   "Submission date": "Submission date",
+  "Support packages": "Support packages",
```

Comment:
Why this translation is in `back` folder? Should be only on `front` folder I guess